### PR TITLE
LessThanPercentage utility script for condition

### PR DESCRIPTION
**Pull request by meirwah**
LessThanPercentage utility script for condition

### CODE REVIEW

**Comment by meirwah** [comment]
@slavikm plz review, not sure on the new format for utility/condition scripts

File: Scripts/script-LessThanPercentage.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+  if (first < second) {
+      return 'LESS';
+  } else {
+      return 'MORE';
+  }
+type: javascript
+tags:
+- Utility
+comment: "Checks if one percentage is less than another.\nReturns LESS: if firstPercentage
```
Comment (slavikm) [inline Scripts/script-LessThanPercentage.yml]: `\n`?

File: Scripts/script-LessThanPercentage.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+commonfields:
+  id: LessThanPercentage
```
Comment (slavikm) [inline Scripts/script-LessThanPercentage.yml]: I would call it LessThan - percentage is not relevant here

File: Scripts/script-LessThanPercentage.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+script: |-
+  first = parseFloat(args.firstPercentage);
+  if (!first) {
+      throw 'firstPercentage is not valid: '+args.firstPercentage;
+  }
+  second = parseFloat(args.secondPercentage);
+  if (!second) {
+      throw 'firstPercentage is not valid: '+args.secondPercentage;
+  }
+  if (first < second) {
```
Comment (slavikm) [inline Scripts/script-LessThanPercentage.yml]: `return first < second ? 'LESS' : 'MORE';`

**Pull request merged**